pkg/connectivity/pod-to-pod.go

```diff
@@ -17,6 +17,7 @@ func PodToPod(fromPod *v1.Pod, toPod *v1.Pod) {
 
 	// TODO
 	meshName := common.MeshName("osm")
+	osmNamespace := common.MeshNamespace("osm-system")
```

Comment:
Can you pass in `settings.Namespace()` from the cli instead? That is set through `osm-health --osm-namespace X`, the same style as the original osm cli.

See example: https://github.com/openservicemesh/osm-health/blob/fae1393387235f838af535c15ba1794646646195/cmd/control_plane_status.go#L17